src/components/film-screen/film-screen.jsx

```diff
@@ -35,14 +35,16 @@ const FilmScreen = (props) => {
   } = props;
 
   const reviewsFilm = findReviews(reviews, name);
-  const genreFilms = filterFilmsByGenre(genre, id);
 
   return (
     <React.Fragment>
-      <section className="movie-card movie-card--full">
+      <section
+        className="movie-card movie-card--full"
+        style={{background: `${backgroundColor}`}}
```

Comment:
Это не относится к нашему курсу, но лучше не использовать комбинированные аттрибуты, если мы задаем только одно значение здесь. То есть лучше так:

```suggestion
        style={{backgroundColor: `${backgroundColor}`}}
```

Reply:
+